server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -2632,15 +2649,39 @@ protected DiskOfferingVO createDiskOffering(final Long userId, final Long domain
         if (bytesReadRate != null && bytesReadRate > 0) {
             newDiskOffering.setBytesReadRate(bytesReadRate);
         }
+        if (bytesReadRateMax != null && bytesReadRateMax > 0) {
```

Comment:
I would recommend avoiding as much as possible any duplicated code. Please call a method such as the proposed method `setIOpsOffering(DiskOfferingV)`.